mos_tests/environment/fuel_client.py

```diff
@@ -175,13 +175,15 @@ def non_primary_controllers(self):
         return non_primary_controllers
 
     def destroy_nodes(self, devops_nodes):
-        logger.info('wait until the nodes get offline state')
+        wait_msg = 'wait until the nodes get offline state'
+        logger.info(wait_msg)
         node_ips = [node.get_ip_address_by_network_name('admin')
                     for node in devops_nodes]
         for node in devops_nodes:
             node.destroy()
         wait(lambda: self.check_nodes_get_offline_state(node_ips),
-             timeout_seconds=10 * 60)
+             timeout_seconds=10 * 60,
+             waiting_for=wait_msg)
```

Comment:
Will raise ' Timeout of 600 seconds expired waiting for wait until the nodes get offline state'
May be better pass 'nodes get offline state' to waiting_for